src/submission/migration/migration-manager.ts

```diff
@@ -257,6 +260,7 @@ export namespace MigrationManager {
       );
 
       const changedValueTypes: string[] = checkValueTypeChanges(
+        diff,
```

Comment:
Have to call `fetchDiff` to get differences of 2 dictionaries because `valueTypeChanges` only contain names of changed fields without info of what's being changed, therefor `valueTypeChanges` alone does not provide enough information to determine if the value type is changed from integer to number.

Reply:
then you can change valueTypeChanges to include what you need, did you try that?